CMakeLists.txt

```diff
@@ -35,11 +36,16 @@ endif()
 add_definitions(-DCOG_INSIDE_COG__=1)
 add_definitions(-DG_LOG_DOMAIN=\"Cog\")
 
+find_package(PkgConfig)
+
+pkg_check_modules(WAYLAND REQUIRED wayland-client)
+if (NOT (WAYLAND_VERSION VERSION_LESS 1.10.0))
+    set(WAYLAND_1_10_OR_GREATER ON)
+endif ()
```

Comment:
Provided that Wayland is required only by the FDO platform module, it seems to me that it would be more correct to move this check inside the `if (COG_PLATFORM_FDO AND NOT COG_USE_WEBKITGTK)` conditional block (below, line ~132).

Reply:
Agree, but empirical trials suggested me that it should go above the `configure_file()` directive below, otherwise cog-config.h will not pick up the correct value of the cmake variable. Any idea on how to work around that?

Reply:
The `configure_file()` call has to be moved to the end of the `CMakeLists.txt` file. That is probably the best location for it anyway, and having it at the end will avoid issues in the future: generating build configuration files should be the last thing done to avoid surprises.

Reply:
Ok, makes sense. I will do that change as part of the new version of this patch.